### PR TITLE
fix: flag list in the experiment creation page shouldn't list archived flags

### LINKED ISSUE #846: fix(ui): flag list in the experiment creation page shouldn't list archived flags

### PR DESCRIPTION
**Pull request by bimalgrg519**
fix #846

### CODE REVIEW

File: ui/web-v2/apps/admin/src/components/ExperimentAddForm/index.tsx
Comment (cre8ivejp) [inline ui/web-v2/apps/admin/src/components/ExperimentAddForm/index.tsx:1]: @bimalgrg519, can you also add to the list of goals?
Not to list archived goals.

Comment (bimalgrg519) [reply ui/web-v2/apps/admin/src/components/ExperimentAddForm/index.tsx:1]: @cre8ivejp, I apologize, but I'm unclear about what you're suggesting to include in the list of goals.

Comment (cre8ivejp) [reply ui/web-v2/apps/admin/src/components/ExperimentAddForm/index.tsx:1]: @bimalgrg519, on the experiment creation page, you list features and goals.
Both lists shouldn't list archived items.

https://github.com/bucketeer-io/bucketeer/pull/851/files#diff-1bfb9b02d93c41cce83a48cb38cfab9fc130e08a74dbeb02603a1b6f5deacaa1R112

**Review by cre8ivejp — APPROVED** [review]
Thanks!

**Pull request merged**